content/collections/destination-catalog/en/bing-ads-cohort.md

```diff
@@ -0,0 +1,83 @@
+---
+id: 8f3c5e7a-9d2b-4f1e-a8c6-3b9d7e5f2a1c
+blueprint: destination-catalog
+use_cases:
+  - "Sending cohorts from Amplitude to Bing Ads enables businesses to target specific groups of users on the Microsoft Advertising Network. By leveraging first-party behavioral data from Amplitude, organizations can create custom cohorts tailored to specific user segments—whether that's high-value customers, returning users, or those at risk of churning—and sync them directly to Bing Ads for precise audience targeting."
+short_description: 'Send Amplitude cohorts to Bing Ads to create customer lists for more personalized campaigns on the Microsoft Advertising Network.'
+integration_category:
+  - ad-networks
+integration_type:
+  - cohorts
+title: "Bing Ads (Cohort Sync)"
+author: 0c3a318b-936a-4cbd-8fdf-771a90c297f0
+connection: destination
+partner_maintained: false
+integration_icon: partner-icons/bing.svg
+exclude_from_sitemap: false
+updated_by: 0c3a318b-936a-4cbd-8fdf-771a90c297f0
+updated_at: 1730332800
+---
+
+Send Amplitude cohorts to Bing Ads to create customer lists for more personalized campaigns. This integration enables you to upload and sync behavioral cohorts directly to the Microsoft Advertising Network for campaign targeting.
+
+## Considerations
+
+- You should have an understanding of Amplitude Activation and behavioral cohorts before setting up this integration.
```

Comment:
Delete "You should have an understanding of Amplitude Activation and behavioral cohorts before setting up this integration."